psyneulink/core/compositions/composition.py

```diff
@@ -10325,16 +10499,16 @@ def _gen_llvm_function(self, *, ctx:pnlvm.LLVMBuilderContext, tags:frozenset):
         else:
             return pnlvm.codegen.gen_composition_exec(ctx, self, tags=tags)
 
-    @handle_external_context(execution_id=NotImplemented)
-    def reset(self, context=None):
-        if context.execution_id is NotImplemented:
-            context.execution_id = self.most_recent_context.execution_id
-
-        self._compilation_data.ptx_execution.set(None, context)
-        self._compilation_data.parameter_struct.set(None, context)
-        self._compilation_data.state_struct.set(None, context)
-        self._compilation_data.data_struct.set(None, context)
-        self._compilation_data.scheduler_conditions.set(None, context)
+    # @handle_external_context(execution_id=NotImplemented)
+    # def reset(self, context=None):
+    #     if context.execution_id is NotImplemented:
+    #         context.execution_id = self.most_recent_context.execution_id
+    #
+    #     self._compilation_data.ptx_execution.set(None, context)
+    #     self._compilation_data.parameter_struct.set(None, context)
+    #     self._compilation_data.state_struct.set(None, context)
+    #     self._compilation_data.data_struct.set(None, context)
+    #     self._compilation_data.scheduler_conditions.set(None, context)
```

Comment:
yeah, this can be dropped. the test was migrated long ago. Not that this is the only way to cleanup binary structures associated with a specific execution id.

Reply:
updated in ffad3efb0747fedd8306bad83a2bfc5dd5ddee29